### PR TITLE
Top level routing stuff

### PR DESCRIPTION
**Pull request by julianjelfs**
and a few fixes

### CODE REVIEW

File: frontend/dev.sh [hunk 1]
```diff
@@ -24,8 +24,8 @@ cd openchat-shared
 pm2 start --name "shared" "npm run dev" 
 cd ../openchat-agent
 pm2 start --name "agent" "npm run dev" 
-cd ../openchat-push
-pm2 start --name "push" "npm run dev" 
+# cd ../openchat-push
```
Comment (hpeebles) [inline frontend/dev.sh:27]: Do you want this included?

File: frontend/openchat-client/src/openchat.ts [hunk 2]
```diff
@@ -4247,9 +4248,9 @@ export class OpenChat extends OpenChatAgentWorker {
         }
     }
 
-    clearSelectedCommunity(): void {
-        selectedCommunityId.set(undefined);
-    }
+    // clearSelectedCommunity(): void {
```
Comment (hpeebles) [inline frontend/openchat-client/src/openchat.ts:4251]: Is this meant to be included?

File: frontend/dev.sh (hunk 1 above)
Comment (julianjelfs) [reply frontend/dev.sh:27]: yes it only affects dev and it just puts a bit less strain on my machine

File: frontend/openchat-client/src/openchat.ts (hunk 2 above)
Comment (julianjelfs) [reply frontend/openchat-client/src/openchat.ts:4251]: Yes - I don't _think_ we need it and it certainly won't cause any immediate problem.

**Review by hpeebles — APPROVED** [review]

**Pull request merged**